stix_shifter_utils/stix_transmission/utils/RestApiClient.py

```diff
@@ -79,6 +79,8 @@ def __init__(self, host, port=None, headers={}, url_modifier_function=None, cert
         self.url_modifier_function = url_modifier_function
         self.auth = auth
 
+        self.logger.error('Deprecated RestApiClient, consider using RestApiClientAsync')
```

Comment:
Does it fail the execution if someone uses it? May be set the logger as WARNING instead?